package.json

```diff
@@ -27,15 +28,26 @@
     "lint": "eslint src test"
   },
   "devDependencies": {
+    "@bigtest/interactor": "^0.9.3",
+    "@bigtest/mirage": "^0.0.1",
+    "@bigtest/mocha": "^0.5.2",
+    "@bigtest/react": "^0.1.2",
     "@folio/eslint-config-stripes": "^5.2.0",
     "@folio/stripes": "^2.12.0",
     "@folio/stripes-cli": "^1.8.0",
     "babel-eslint": "^9.0.0",
-    "eslint": "^5.5.0"
+    "babel-polyfill": "^6.26.0",
```

Comment:
Definitely don't use babel-polyfill, since it'll bring an old copy of Babel into your workspace. And also don't use @babel/polyfill, since [that's been deprecated](https://babeljs.io/docs/en/babel-polyfill).

Reply:
Looks like that came in via the `stripes app bigtest` that I used to setup the BigTests. Will take it off.